### ISSUE TITLE
R&D Review - Phase 1 

### ISSUE THREAD

**Issue by misterinterrupt** [misterinterrupt/neohexane#7]
Catalog; Documentation; Tech Spec; etc

**Comment by misterinterrupt** [comment]
8n progress, working in evernote to collect past milestones

**Comment by misterinterrupt** [comment]
I decided to design a synth made to be sequenced by a drum machine or other midi sequencer. 

I designed the signal path for 6 different weird drum synths in the vein of Elektron's Rytm and Nord's Drum. The idea of the end product was somewhere between those and the, then yet to be released, Korg Volca Drum. 

The design was focused on a synth that made the cannonical 808 sounds but also had a multidimensional twist, some level of comb filters and flanging would be built into the actual signal path.

I made a simple synth with the teensyduino audio library. then went on to take time to fix a bug in the amplitude envelope and Iearn a little about how the teensy audio lib works. 

The next step for me was to begin building  to I ended up not being into the idea that i would spend so much time learning c++ instead of designing a synth.

Found the FAUST DSP language and compiler ecosystem and decided that this was perfect for my level of design and understanding. It would allow me to be fully creative without slowing me down with language details like C++ might. Some amount of C++ is still required, but only for parts of the software that are not synthesis related. I am very happy with this arrangement.

Came to a place in development where I realized that the synth did not need separate synths for each hit, I could make a preset library that represented what one signal path could do, from the classic 808/X0X/etc. drum sounds to noise/drone sounds.

Neohexane became a synth capable of noise and percussion sounds. it is set up to be six voice multitimbral, each voice is 1 note monophonic, but as a stretch goal, each single voice may be playable chromatically.
